### ISSUE TITLE
autogen.sh uses bash builtin

### ISSUE THREAD

**Issue by YarNhoj** [orkmellanox/sockperf#63]
The autogen.sh script is using a bash builtin to source /build/versioning.sh. This is fine so long as /bin/sh is linked to bash but on my debian distro /bin/sh -> dash, which doesn't support the source builtin. Probably want to change this to just dot. 

```
CURRENT_VERSION_FILE=./build/current-version
if [ ! -f $CURRENT_VERSION_FILE ] # don't ruin file that might come from build-git-rpm.sh
then
        source ./build/versioning.sh #<-change this line? . ./build/versioning.sh
        echo ${VERSION}-${VER_GIT} > $CURRENT_VERSION_FILE
        echo $GIT_REF >> $CURRENT_VERSION_FILE
fi
```

**Comment by avnerbh** [comment]
Thanks. I will change it to dot tomorrow (Thursday)
Avner

בתאריך יום ד׳, 14 באוק׳ 2015, 23:40 מאת John Ray notifications@github.com:

**Comment by avnerbh** [comment]
fixed in commit bdaca39e31a2e6ca601ee6c398bec0c95e89c512
